pkg/operator/ceph/cluster/crash/reconcile.go

```diff
@@ -192,6 +193,35 @@ func (r *ReconcileNode) reconcile(request reconcile.Request) (reconcile.Result,
 			}
 			logger.Debugf("deployment successfully reconciled for node %q. operation: %q", request.Name, op)
 		}
+
+		// ceph crash prune configuration
+		if cephCluster.Spec.CrashCollector.DaysToRetain == 0 {
```

Comment:
This method is getting long, how about factoring the retention policy out to another method?